omc3/correction/handler.py

```diff
@@ -216,9 +280,9 @@ def _maybe_add_coupling_to_model(model: tfs.TfsDataFrame, keys: Sequence[str]) -
     return model
 
 
-def _create_corrected_model(twiss_out: Path | str, change_params: Sequence[Path], accel_inst: Accelerator) -> tfs.TfsDataFrame:
+def _create_corrected_model(twiss_out: Path | str, change_params: Sequence[Path], accel_inst: Accelerator, update_dpp: bool = False) -> tfs.TfsDataFrame:
```

Comment:
Same as below, surely `update_dpp` here isn't expected to be a bool, considering the later checks and value passed to a function which expects `float | str`.

Reply:
See above